app/src/main/assets/viewer.js

```diff
@@ -108,6 +116,18 @@ function renderPage(pageNumber, zoom, prerender, prerenderTrigger=0) {
             return;
         }
 
+        const defaultZoomRatio = getDefaultZoomRatio(page, orientationDegrees);
+
+        if (resetZoomRatio) {
+            zoomRatio = defaultZoomRatio
+            newZoomRatio = defaultZoomRatio;
+            channel.setZoomRatio(defaultZoomRatio);
```

Comment:
From a consistency standpoint it would (imo) make sense to reset after hitting the "rotate clockwise/counterclockwise" button, because when actually rotating the whole device it would be reset too because the whole webview gets recreated.

Reply:
It makes sense to preserve zoom ratio (set by the user). Also, webview shouldn't need to reload after device is rotated, it is a different issue.

Reply:
Ah, I just assumed the webview reload was intentional (maybe it would add a lot of complexity to handle the whole change or something)

I removed the part which reset the zoom ratio after changing orientation, and I also removed the `resetZoomRatio` variable because the first document load can also be detected by checking if the cache is empty.